src/pages/survey/answer/[surveyId]/index.tsx

```diff
@@ -89,6 +91,7 @@ function SurveyResultsPage({
             {surveyData?.title}
           </h1>
           <div className="flex w-full justify-center gap-2 sm:w-auto">
+            {surveyData?.userId===userId && typeof surveyData?.isActive==='boolean' && <Toggle isEnabled={surveyData?.isActive || false} onToggle={updateSurveyStatus} label={t('toggleShow')} />}
```

Comment:
The page is only available for survey's owner so `surveyData?.userId===userId && typeof surveyData?.isActive==='boolean'` is not necessary

Reply:
`surveyData?.isActive || false` can be shorten to `!!suveyData?.isActive`